azurerm/resource_arm_mssql_elasticpool.go

```diff
@@ -176,78 +176,165 @@ func resourceArmMsSqlElasticPool() *schema.Resource {
 		CustomizeDiff: func(diff *schema.ResourceDiff, v interface{}) error {
 
 			name, _ := diff.GetOk("sku.0.name")
+			tier, _ := diff.GetOk("sku.0.tier")
 			capacity, _ := diff.GetOk("sku.0.capacity")
+			family, _ := diff.GetOk("sku.0.family")
+			maxSizeGb, _ := diff.GetOk("max_size_gb")
```

Comment:
imo these fields are likely to be interpolated, so bear in mind the `customizeDiff` function will return empty/default values for these fields if they're interpolated